src/Setup.php

```diff
@@ -97,14 +97,35 @@ public function bcgov_block_theme(): void {
     public function set_javascript_variables() {
 
 		$javascript_variables = [
-			'domain'          => home_url(),
-			'siteName'        => esc_attr( get_option( 'active_site_pattern_styles' ) ),
-			'templateDir'     => get_template_directory_uri(),
-			'headerEffect'    => esc_attr( get_option( 'header_effect' ) ),
-			'allSiteStyles'   => esc_attr( get_option( 'enable_all_styles' ) ),
-			'customBodyClass' => esc_attr( get_option( 'custom_body_class' ) ),
+			'domain'                  => home_url(),
```

Comment:
is this in the global namespace, or define in some other name space, it is global we should be cautious, as there is more chance of potential collisions.